recipe/meta.yaml

```diff
@@ -14,7 +14,7 @@ source:
     - 0001-remove-wil-from-exported-config.patch
 
 build:
-  number: 0
+  number: 1
```

Comment:
I don't think there is any need to upload new binaries for this change. We are simply dropping build variants. Bumping the build number to 1 will cause an identical libcurl8/openssl3 variant to be uploaded to anaconda.org